bundle/src/files.rs

```
@@ -48,10 +48,17 @@ impl FileSetBuilder {
                 .iter()
                 .cloned()
                 .map(|mut junit_wrapper| {
-                    junit_wrapper.junit_path = PathBuf::from(junit_wrapper.junit_path)
-                        .join("**/*.xml")
-                        .to_string_lossy()
-                        .to_string();
+                    if junit_wrapper.junit_path.ends_with(".xml") {
+                        junit_wrapper.junit_path = PathBuf::from(junit_wrapper.junit_path)
+                            .join("**/*.xml")
+                            .to_string_lossy()
+                            .to_string();
+                    } else {
```

Since the body of this block is assuming an extension of `.bin`, should we be if-elsing on `.bin` with an error (or log) on a catchall `else`? (Also applies for the if/else on line 116 this file.

---

Yeah, good call. Let's throw an error.